ReactNativeUiLib.podspec

```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = "ReactNativeUiLib"
+  s.version      = package['version']
+  s.summary      = "React Native UI Library"
+
+  s.authors      = "Wix.com"
+  s.homepage     = "https://github.com/wix/react-native-ui-lib"
+  s.license      = package['license']
+  s.platforms    = { :ios => "9.0", :tvos => "9.2" }
+
+  s.module_name  = 'ReactNativeNavigation'
+
+  s.source       = { :git => "https://github.com/wix/react-native-ui-lib.git", :tag => "#{s.version}" }
+  s.source_files  = "lib/ios/**/*.{h,m}"
```

Comment:
Path is wrong

Reply:
Actualy it is correct.  The top-level `ios/` path is for a demo app.